src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,269 @@
+        filename_estimated_cn: str,
+        filename_mediancov: str,
+        filename_wgd: str,
+        filename_cnv_qvalues: str,
+        filename_high_manta: str,
+        filename_high_melt: str,
+        filename_high_wham: str,
+        filename_low_manta: str,
+        filename_low_melt: str,
+        filename_low_wham: str,
+        filename_melt_insert_size: str,
+        output_prefix: str) -> None:
+    """
+    Reads the provided TSV files (tab-delimited) and merges all the information in one table
+    serialized to the given output filename.
+    """
+    df_ploidy = read_ploidy(filename_estimated_cn)
+    df_bincov_median = read_bincov_median(filename_mediancov)
+    df_wgd_scores = read_wgd_scores(filename_wgd)
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"))
+    df_total_high_outliers = read_all_outlier(df_manta_high_outlier, df_melt_high_outlier, df_wham_high_outlier, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"))
+    df_total_low_outliers = read_all_outlier(df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, "low")
+    df_melt_insert_size = read_melt_insert_size(filename_melt_insert_size)
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers,
+           df_melt_insert_size]
+    output_df = reduce(lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs)
+    output_df = output_df[output_df[ID_COL] != EMPTY_OUTLIERS]
+
+    # save the file
+    output_df.to_csv(f"{output_prefix}.evidence_qc_table.tsv", sep="\t", header=True, index=False, na_rep=np.nan)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-y", "--estimated-copy-number-filename",
+        help="Sets the filename containing estimated copy numbers per contig.")
+
+    parser.add_argument(
+        "-d", "--median-cov-filename",
+        help="Sets the filename containing median coverage.")
+
+    parser.add_argument(
+        "-g", "--wgd-scores-filename",
+        help="Sets the filename containing WGD scores.")
+
+    parser.add_argument(
+        "-b", "--binwise-cnv-qvalues-filename",
+        help="Sets the filename containing bin-wise CNV q-values.")
+
+    parser.add_argument(
+        "-z", "--manta-qc-outlier-high-filename",
```

Comment:
Not necessary to merge: Scramble outlier files are missing here, so we should make sure to come back and update this script to handle them later. 
In the mean time, please remove the scramble inputs in the WDL task since the script does not use them.